centos_playground/onboard/onboard.py

```diff
@@ -0,0 +1,53 @@
+import os
+from git import Repo
+from ogr.services.pagure import PagureService
+from ogr.abstract import AccessLevel
+
+from survey import CentosPackageSurvey
+
+
+service = PagureService(token=os.getenv('PAGURE_TOKEN'),
+                        instance_url='https://git.stg.centos.org/')
+
+
+class OnboardCentosPKG():
+    def __init__(self, pkg_name):
+        self.pkg_name = pkg_name
+        self.cps = CentosPackageSurvey({
+            'fullname': f'rpms/{pkg_name}',
+            'name': pkg_name,
+        })
+
+    def run(self):
+        p = service.get_project(namespace='source-git', repo=self.pkg_name)
+        if p.exists():
+            print(f'Source repo for {self.pkg_name} already exists')
+            return
+        self.cps.run()
+        if self.cps.pkg_res.get('error') or self.cps.pkg_res.get('conditional_patch'):
+            print(f'Onboard aborted for {self.pkg_name}:')
+            print(self.cps.pkg_res)
+            return
+        print('Convert result:')
+        print(self.cps.pkg_res)
+        p = service.project_create(
+            repo=self.pkg_name,
+            namespace='source-git',
+            description=f"Source git repo for {self.pkg_name}.\n"
+                        f"For more info see: http://packit.dev/docs/source-git/")
+
+        p.add_user('centosrcm', AccessLevel.maintain)
+        p.add_group('git-packit-team', AccessLevel.maintain)
+        gr = Repo(self.cps.src_dir)
+        gr.create_remote('packit', p.get_git_urls()['ssh'])
+        gr.git.push('packit', 'c8', tags=True)
```

Comment:
You came from the same area as Tomáš, didn't you?...;)

Please, don't use the cryptic variable names or at least don't reuse them...

Reply:
gr = git repo, pretty clear to me -_- :D

Reply:
Yes, `self.cps.pkg_res.get('conditional_patch')` sounds very clear..;)